.github/workflows/client.yml

```diff
@@ -49,76 +49,37 @@ jobs:
     runs-on: ubuntu-18.04
     needs: build
     if: github.ref == 'refs/heads/master'
+    strategy:
+      matrix:
+        environment:
+          [
+            { "name": "ambroise", "secret": "IP_VM_AMB" },
+            { "name": "aphp", "secret": "IP_VM_AP" },
+            { "name": "clb", "secret": "IP_VM_CLB" },
+            { "name": "curie", "secret": "IP_VM_CURIE" },
+            { "name": "demo", "secret": "" },
+            { "name": "dev", "secret": "" },
+            { "name": "rms", "secret": "IP_VM_REIMS" },
+            { "name": "staging", "secret": "" },
+          ]
     steps:
       - uses: actions/checkout@v1
 
-      - name: Build staging and demo
-        run: |
-          cd client/
-          yarn build:docker:staging
-          yarn build:docker:dev
-          yarn build:docker:demo
```

Comment:
Shouldn't we keep those? or add IP_STAGING and IP_DEMO in the strategy?

Reply:
Oops, indeed I need to add IP_staging, etc. in the GH secrets if I want to do that
Which makes me even less convinced that this is the best solution 😅

Reply:
good catch @elsiehoffet-94. I don't think it's that bad to add the url of our environments in GH secrets BUT if you're not convinced there is an other solution:
- keep staging,demo,dev in the matrix and remove the "secret" field (or keep in empty)
- do not generate a .env.<target> file for environments which do not have the "secret" field in the matrix --> use the .env.[dev|staging|demo] that we push in the repo instead